sandbox/index.html.example

```diff
@@ -4,19 +4,25 @@
   <meta charset="utf-8" />
   <title>Video.js Sandbox</title>
 
-  <link href="../build/files/video-js.css" rel="stylesheet" type="text/css">
+  <link href="../build/temp/video-js.css" rel="stylesheet" type="text/css">
 
   <!-- LOAD VIDEO.JS SOURCE FILES IN ORDER -->
-  <script src="../build/source-loader.js"></script>
+  <script src="../build/temp/video.js"></script>
```

Comment:
Right now Chrome supports a good deal of what we've already implemented, but I don't think any browsers support ES6 modules, so Browserify is still a hard requirement.